accuracy/Hyperboloid.qmd

```diff
@@ -0,0 +1,86 @@
+---
+title: Accurcay tests on the Hyperboloid
+---
+
+
+```{julia}
+#| echo: false
+#| code-fold: true
+#| output: false
+using Pkg;
+cd(@__DIR__)
+Pkg.activate("."); # for reproducibility use the local accuracy environment.
+```
+
+In this small note we compare different implementations of several functions on the [Hyperboloic]().
+We compare the functions prior [PR #]() to vs those introduced therein.
+We compare the previous version (always called `_1`) and the new version `_2`, where we leave out the manifold to indicate, that these are the local functions here.
+We compare this to the one given in `Manifolds.jl` at time of writing (which is the first one), on `BigFloat` on several scales of magnitudes.
+
+```{julia}
+#| output: false
+using Manifolds, Random, NamedColors, Plots
+using Manifolds: minkowski_metric
+
+paul_tol = load_paul_tol()
+indigo = paul_tol["mutedindigo"]
+teal = paul_tol["mutedteal"]
+```
+
+## Reference Data
+
+We choose a “base point”, a direction and create tangent vectors and correcponding points in several magnutudes of distances
+```{julia}
+#| output: false
+M = Hyperbolic(2)
+p = [0.0, 0.0, 0.1]
```

Comment:
This is not a valid point on `M`.

Reply:
I've re-run tests with ` p = [0.0, 0.0, 1.0]` and it looks like the larger error of `distance_2` for larger `Xs` coincides with `qs` being inaccurate themselves (`check_point` returns a huge error). So I'd say we can just use `distance_2`.

Reply:
ups. Yes, That is what I concluded when I first compared to distance with BigFloat and not with norm. Note that the exp is also the old (asinh) one not the new acosh).

Reply:
I'd also conclude that the new distance (2) is the one we should use, but it was still good to check that.

Reply:
Yes, I think we should go with this change. Now we are sure this new approach is more accurate.

Reply:
Hajg has already also adapted PT, maybe just not yet pushed here and I will already now accept your changes :)